src/ast/node/function.rs

```diff
@@ -185,7 +185,12 @@ impl FuncCallNode {
     fn build_hint(&mut self, ctx: &mut Ctx, fnvalue: &FNValue, skip: u32) {
         for (i, para) in self.paralist.iter_mut().enumerate() {
             let pararange = para.range();
-            ctx.push_param_hint(pararange, fnvalue.param_names[i + skip as usize].clone());
+            let mut b = FmtBuilder::default();
+            para.format(&mut b);
+            let para_s = b.generate();
```

Comment:
can consider to rename it as `param_name` or something else. I'm not sure what's the `_s` means

Reply:
it means str